product_attribute_company_favorite/tests/test_product_attribute.py

```diff
@@ -0,0 +1,165 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestProductAttributeMultiCompany(TransactionCase):
```

Comment:
Should probably be SavepointCase in Odoo 14, TransactionCase in Odoo ≥15

Reply:
I don't understand, this code is for Odoo 16 so it's OK no ?

Reply:
Oh it is! I was working on Odoo 14 code at the time I reviewed this; never mind this comment.